linters/ruby/.rubocop.yml

```diff
@@ -0,0 +1,50 @@
+AllCops:
+  Include:
+    - "**/Rakefile"
+    - "**/config.ru"
+  Exclude:
+    - "db/**/*"
+    - "config/**/*"
+    - "script/**/*"
+    - "bin/**/*"
+    - "vendor/**/*"
+    - "node_modules/**/*"
+    - "spec/factories.rb"
+
+Metrics/LineLength:
+  Enabled: false
+
+Style/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
+Style/ClassAndModuleChildren:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/DotPosition:
+  EnforcedStyle: trailing
+
+Style/IfUnlessModifier:
+  Enabled: false
+
+Style/PredicateName:
+  NamePrefixBlacklist:
+    - is_
+    - have_
+
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
+
+Style/NumericLiterals:
+  Enabled: false
+
+Style/SingleLineBlockParams:
+  Enabled: false
+
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Metrics/MethodLength:
+  Enabled: false
```

Comment:
I think I had some situation where this was useful, but we can remove make
it true as a default.
On Sat, 9 Jan 2016 at 00:37, Miguel Palhas <notifications@github.com> wrote:

> In linters/ruby/.rubocop.yml
> <https://github.com/subvisual/guides/pull/9#discussion_r49255339>:
>
> > +    - have_
> > +
> > +Style/StringLiterals:
> > +  EnforcedStyle: double_quotes
> > +
> > +Style/NumericLiterals:
> > +  Enabled: false
> > +
> > +Style/SingleLineBlockParams:
> > +  Enabled: false
> > +
> > +Style/MultilineOperationIndentation:
> > +  EnforcedStyle: indented
> > +
> > +Metrics/MethodLength:
> > +  Enabled: false
>
> what's the reason for disabling this last one?
> I don't remember ever using it, and I think I actually prefer to have it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/subvisual/guides/pull/9/files#r49255339>.
>